model/events/LtiAgsListener.php

```diff
@@ -81,19 +83,32 @@ public function onDeliveryExecutionStateUpdate(DeliveryExecutionState $event)
         }
     }
 
+    public function onDeliveryExecutionResultsRecalculated(DeliveryExecutionResultsRecalculated $event): void
```

Comment:
I believe that we should update the composer.json dependency `oat-sa/extension-tao-outcome` after you release the PR [#232](https://github.com/oat-sa/extension-tao-outcome/pull/232). Does it make sense?

Reply:
Sure, and https://github.com/oat-sa/extension-tao-lti/pull/361 as well